src/components/PokemonType.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { Text, useTheme } from 'react-native-paper';
+
+export default function PokemonType({ passedStyle, type }) {
+  const colors = useTheme().colors;
+  const typeColor = colors.pokemon.types[type];
+  return (
+    <View style={[styles.container, { backgroundColor: typeColor }, passedStyle]}>
+      <Text style={{ fontSize: 12, color: colors.white }}>{type.toUpperCase()}</Text>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
```

Comment:
In my opinion it would be better if every type had the same shape - it is worth to add 'width' attribute to this compotent.

Reply:
Fixed
<img src=https://user-images.githubusercontent.com/46849151/116210693-63909800-a743-11eb-9a66-c5ba85b21845.png width=400/>